libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/actionbutton/ActionButtonBinder.kt

```diff
@@ -33,7 +33,7 @@ import com.mapbox.navigation.ui.voice.view.MapboxAudioGuidanceButton
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.flow.StateFlow
 
-@ExperimentalPreviewMapboxNavigationAPI
+@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
```

Comment:
This uses `MapboxExtendableButton` which is still `@Experimental`. 

> not removing @ExperimentalPreviewMapboxNavigationAPI from MapboxExtendableButton and it's usage across the SDK. This is because in some cases it's still not possible to render circular buttons with this widget. Issue was reported https://github.com/mapbox/mapbox-navigation-android/issues/6248